src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/InterceptedProjectProperties/ApplicationPropertyPage/ApplicationFrameworkValueProvider.cs

```diff
@@ -318,6 +318,27 @@ private async Task<string> GetPropertyValueAsync(string propertyName)
                     _ => unevaluatedPropertyValue
                 };
             }
+            else if (propertyName == MinimumSplashScreenDisplayTime)
+            {
+                if (int.TryParse(unevaluatedPropertyValue, out int value))
+                {
+                    if (value < 0)
```

Comment:
If the RegEx is working, I don't think you could get a negative value here (unless the property can be set elsewhere, where our UI validation doesn't apply.)

Not a big deal to leave this as-is however.

Reply:
I'll leave it for now as it was an easy thing to change and ensure the user input is valid, but if we have a strong feeling that we should stick to the RegEx validation, I'm happy to remove it. :)